learning/features/assetlib/uploading_to_assetlib.rst

```diff
@@ -0,0 +1,105 @@
+.. _doc_uploading_to_assetlib:
+
+Submitting to the Asset Library
+===============================
+
+Introduction
+------------
+
+This tutorial aims to serve as both a guide on how you can submit your own assets
+to the Godot Asset Library and share them with the Godot community.
+
+As mentioned in the :ref:`doc_using_assetlib` document, in order to be able to
+submit assets to the AssetLib, you need to have a registered account, and be
+logged in.
+
+Submitting
+----------
+
+Once you are logged in, you will be able to head over to the "Submit Assets" page
+of the AssetLib, which will look like this:
+
+|image0|
+
+While it may look like a lot (and there is more to scroll down), each field is
+described in terms of what you should put in. We will nonetheless go over what
+is required in the submission form here as well.
+
+* **Asset Name**: 
+    The name of your asset. Should be a unique, descriptive title of 
+    what your asset is.
+* **Category**: 
+    The category that your asset belongs to, and will be shown in
+    search results. The category is split into **Addons** and **Projects**. Assets
+    of the Project type (Templates, Demos, Projects) do not show up in the in-editor
+    AssetLib, and can only be downloaded from the website version of the AssetLib,
+    as well as from the provider directly (see below).
+* **Godot version**: 
+    The version of the engine that the asset works with.
+    Currently it's not possible to have a single asset entry contain downloads for
+    multiple engine versions, so you may need to re-submit the asset mulitple times,
+    with an etry for each Godot version it supports. This is particularly important
+    when dealing with major versions of the engine, such as Godot 2.x and Godot 3.x.
+* **Version**: 
+    The version number of the asset. While you are free to choose
+    and use any  versioning scheme that you like, you may want to look into
+    something such as `SemVer <https://semver.org>`_ if you want your asset's
+    versioning scheme to be clear and consistent. 
```

Comment:
Note that there is also an internal version number, which is incremented every time the asset's download link is updated (or so).